managed/services/telemetry/config.default.yml

```diff
@@ -544,3 +544,17 @@ telemetry:
     data:
       - metric_name: "k8s_clusters_count"
         column: "count"
+
+# MySQL Clusters
+  - id: MySQLClustersCount
+    source: PMMDB_SELECT
+    query: aggregated_nodes.nodes_count as nodes_count, COUNT(*) as clusters_count FROM (SELECT cluster, replication_set, COUNT(*) as nodes_count FROM services WHERE service_type = 'mysql' GROUP BY cluster, replication_set) as aggregated_nodes GROUP BY aggregated_nodes.nodes_count
```

Comment:
Does it work for PXC clusters and Group Replication, right? Should we distinguish whether it's PXC cluster or Group Replication?

Reply:
Discussed verbally with @rnovikovP  - will try to rework to fetch data from VM as there we have more consistent data than in PG replication names/ cluster names